Infrastructure/GhostNetwork.Content.MongoDb/MongoCommentsStorage.cs

```diff
@@ -49,6 +49,22 @@ public async Task<string> InsertOneAsync(Comment comment)
             return entity.Id.ToString();
         }
 
+        public async Task<(Comment, int)> UpdateOneAsync(string commentId, string content)
+        {
+            if (!ObjectId.TryParse(commentId, out var oId))
+            {
+                return (null, default(int));
+            }
+
+            var filter = Builders<CommentEntity>.Filter.Eq(x => x.Id, oId);
+            var update = Builders<CommentEntity>.Update.Set(x => x.Content, content);
+
+            var updateResult = await context.Comments
+                .UpdateOneAsync(filter, update);
+
+            return (await FindOneByIdAsync(commentId), (int)updateResult.ModifiedCount);
```

Comment:
`Update**One**Async` should update only ONE document in database, so what is the reason of returning `ModifiedCount`

I would say there is no reason to return even comment because you can do in directly in service or controller if needed

Reply:
Removed it, that was made for check the same comment content like was in db.